nutrition_labels/grant_tagger.py

```diff
@@ -83,7 +89,7 @@ def transform(self, data):
 
 
         X_train, X_test, y_train, y_test = train_test_split(X_vect, y, test_size=self.test_size,
-                                                            random_state=42)
+                                                            random_state=self.random_state)
```

Comment:
this might be being to pernickety but do you think it is worth setting two random states so we can see the affect of different train/test splits on the same data set. Might not be worth the complication please feel free to ignore

Reply:
no you're right, good point. I'll add another variable.

Reply:
changes it in c9ce4a51fc3b03ae0e05cc06d7adfea9da377769